apps/coscrad-frontend/src/components/coscrad-byline/coscrad-byline.tsx

```diff
@@ -1,10 +1,31 @@
+import { Grid, Typography } from '@mui/material';
 import { COSCRADLogo } from '../coscrad-logo/coscrad-logo';
-import './coscrad-byline.css';
 
 export const COSCRADByline = (): JSX.Element => {
     return (
-        <span className="coscrad-byline">
-            A project built on the <COSCRADLogo /> platform.
-        </span>
+        <Grid
+            container
+            direction="row"
+            justifyContent="center"
+            alignItems="center"
+            sx={{
+                mt: 2,
+                width: '100%',
+            }}
+        >
+            <Grid item>
```

Comment:
This is kind of funky. My intuition says there should be a `<GridItem ... />`. But not our API.

Reply:
`item` is part of the MUI `Grid` API, just as you can create a `<Grid container ...>`